test/Tester/OrleansLoggingTests.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Orleans.Extensions.Logging;
+using Orleans.Runtime;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using Orleans.Extensions.Logging.Legacy;
+using Xunit;
+
+namespace Tester
+{
+    [TestCategory("BVT"), TestCategory("OrleansLogging")]
+    public class OrleansLoggingTests
+    {
+        [Fact]
+        [Obsolete]
+        public void OrleansLoggingCanConfigurePerCategoryServeriyOverrides()
+        {
+            //configure logging with severity overrides
+            IServiceCollection serviceCollection = new ServiceCollection();
+            serviceCollection.AddLogging();
+            var severityOverrides = new OrleansLoggerSeverityOverrides();
+            severityOverrides.LoggerSeverityOverrides.Add(this.GetType().FullName, Severity.Warning);
+            serviceCollection.AddLogging(builder => builder.AddLegacyOrleansLogging(new List<ILogConsumer>()
+            {
+                new LegacyFileLogConsumer($"{this.GetType().Name}.log")
+            }, severityOverrides));
+            var serviceProvider = serviceCollection.BuildServiceProvider();
```

Comment:
Why is DI involved in these tests? I see no need or value to it.

Reply:
the runtime gets logger from DI. I want to emulate that